### ISSUE TITLE
Event text is cut off

### ISSUE THREAD

**Issue by alexanderdd** [nextcloud/calendar#3274]
### Steps to reproduce
1. Have time steps of 30mins
2. create two events, where the end time of the first is the start time of the second

### Expected behaviour
Event text should be readable for all events

### Actual behaviour
Event text of first event is cut off:
![120724864-f0faa100-c4d4-11eb-83e6-9d8649631860](https://user-images.githubusercontent.com/28591861/124033316-1ad2b500-d9fa-11eb-90f6-91674f18d6b4.png)

This is **related** but not equal to https://github.com/nextcloud/calendar/issues/2644 which was fixed by https://github.com/nextcloud/calendar/pull/3121

### Calendar app
**Calendar app version:** 2.3.0

### Client configuration
**Browser:** Firefox 89.0.2

**Operating system:** LinuxMint

### Server configuration

**Nextcloud Version:** 20.0.10

......................Please ask if you need more details.

Edit: adding relevant keywords: overlap overlapping

**Comment by alexanderdd** [comment]
still present in calendar 2.3.1

@jonathantreffler @georgehrke please triage. This is a **serious** UX issue, interface not usable! You guys never have events that touch?

**Comment by szaimen** [comment]
Still valid?

**Comment by alexanderdd** [comment]
it's ok now.

**Comment by ChristophWurst** [comment]
well you're certainly welcome @alexanderdd!

**Comment by alexanderdd** [comment]
I*m afraid I have to open this again. Nextcloud 22.2.3 calendar  3.0.2
![Bildschirmfoto vom 2021-12-21 10-33-08](https://user-images.githubusercontent.com/28591861/146906761-009f9219-dd7b-4382-af22-7e7790e7d53f.png)

**Comment by alexanderdd** [comment]
@szaimen or @ChristophWurst can you reproduce?

**Comment by alexanderdd** [comment]
This is still an issue in NC 30, on c.nc.com, cannot check app version:

30mins timesteps: ok
![grafik](https://github.com/user-attachments/assets/ac6e1d21-4e11-4b24-93db-a5b2fa05b58d)

20mins timesteps: Not ok
![grafik](https://github.com/user-attachments/assets/04628dff-8219-4f70-9bf9-606e4e74bfa1)

15min timesteps: ok
![grafik](https://github.com/user-attachments/assets/909a45ca-adda-4bf7-91cd-8395233899f7)

**Comment by miaulalala** [comment]
Can you check with the latest release @alexanderdd ?

This is what it looks like for me today:
![image](https://github.com/user-attachments/assets/5d3c5341-4a1d-476f-afa6-88e8f3d7a6be)
![image](https://github.com/user-attachments/assets/cd1e5e7a-2cf5-4f8d-9947-bdac7b087485)

**Comment by szaimen** [comment]
I just tested on c.nc.c and this specific case does indeed not work yet: 
![image](https://github.com/user-attachments/assets/4a37daae-0608-48a7-b58f-57ef31245d00)

**Comment by miaulalala** [comment]
Can reproduce as well: 
![image](https://github.com/user-attachments/assets/a1c32ef3-4df7-40a2-883c-d0a009695e74)

**Comment by miaulalala** [comment]
I think this might be this css rule:

![image](https://github.com/user-attachments/assets/80d946ea-236b-4a7d-9c25-084d6379165d)

Disabling the 4em min height that also places the 15- minute offset more visually accurate on the grid:

![image](https://github.com/user-attachments/assets/4d4f25fb-ba12-441d-bdd5-dbf0d998f6fc)

**Comment by alexanderdd** [comment]
Problem still present, this is right now on c.nc.com
![Image](https://github.com/user-attachments/assets/d13a99d6-4b91-4645-be47-dcb8a8cedfc3)

Note that it is not only when events touch.

**Comment by jancborchardt** [comment]
Both Google Calendar and MacOS Calendar show the title first and the times only after. Since the times already have a visual representation in the calendar, that's a better way to display it.

If somehow this is not possible with Fullcalendar, alternatively I would suggest:
- Only showing the start time, omitting the end time
- Always starting the title directly after the time, not only when there is little space
(This is e.g. how Business Calendar 2 on Android does it.)

**Comment by alexanderdd** [comment]
@jancborchardt this issue here is about event title being cut off. See my comment before yours.

I also think it is good to show the title before the time, and if there is not enough space, only show the title -- but that should be a **separate issue?** not sure if one already exists?

**Comment by joshtrichards** [comment]
Related: #3521

**Comment by alexanderdd** [comment]
this is still an issue on c.nc.com right now, NC "32.0.0 RC3"
cannot check calendar version
